okhttp/src/main/java/io/grpc/okhttp/OkHttpClientTransport.java

```diff
@@ -480,9 +480,10 @@ int getPendingStreamSize() {
   /**
    * Finish all active streams due to an IOException, then close the transport.
    */
-  void onException(Throwable failureCause) {
-    log.log(Level.WARNING, "Transport failed", failureCause);
-    startGoAway(0, Status.UNAVAILABLE.withCause(failureCause));
+  void onException(Throwable cause) {
+    log.log(Level.WARNING, "Transport failed", cause);
+    startGoAway(0, Status.UNAVAILABLE.withCause(cause)
+        .augmentDescription(cause == null ? "No provided cause" : cause.getMessage()));
```

Comment:
We've not been tending to copy the exception message to the status message. We can discuss doing that, but for the sake of the original issue that doesn't aid because it doesn't add additional information.